release-plz.toml

```diff
@@ -0,0 +1,102 @@
+[workspace]
+
+changelog_update = true
+
+dependencies_update = true
+
+git_tag_enable = true
+
+git_release_enable = false
+
+pr_labels = ["release"]
+
+allow_dirty = false
+
+publish_allow_dirty = false
+
+semver_check = true
```

Comment:
This file is stolen from the Edge release-plz.toml. This is the only meaningful change I've made here because semver checks do actually make sense for Ygg